package.json

```diff
@@ -21,5 +21,8 @@
   "workspaces": [
     "apps/*",
     "packages/*"
-  ]
+  ],
```

Comment:
This shouldn't be here, it needs to be in the package.json of the UI (I believe there it is already there)

Reply:
I've deleted it